guides/installing-python.md

```diff
@@ -0,0 +1,89 @@
+# Installing and using Python 2 and 3
+
+## Remove Homebrew Python, virtualenvwrapper, and variables
+
+Before installing and configuring pyenv, uninstall all old versions of Python that might be installed via Homebrew and virtualenvwrapper, and remove any varaibles from your shell's dotfiles (`.bashrc` for bash users, `.zshrc` for zsh users):
+
+```shell
+brew uninstall python
+brew cleanup python
+```
```

Comment:
We should add a note here about what to do if you have things installed via Homebrew that depend on Python (basically, make sure you're adding homebrew to `PATH` before you `eval "$(pyenv init -)"`)